OpenDreamRuntime/Objects/DreamObject.cs

```diff
@@ -29,7 +29,7 @@ public class DreamObject {
             thread.PushProcState(procState);
 
             if (thread.Resume() == DreamValue.Null) {
-                throw new InvalidOperationException("DreamObject.InitSpawn called a yielding proc!");
+                throw new InvalidOperationException($"DreamObject.InitSpawn ({this.ObjectDefinition.Type}) called a yielding proc!");
```

Comment:
It would be better to include a DM stack trace. This can be done by calling `thread.HandleException`
```suggestion
                Exception exception = new InvalidOperationException("DreamObject.InitSpawn called a yielding proc!");

                thread.HandleException(exception);
                throw exception;
```
Though I'm not sure if this should throw. Execution should be able to continue despite this.

Reply:
So what should I do?

Reply:
Just call `thread.HandleException`, don't throw.
```suggestion
                thread.HandleException(new InvalidOperationException("DreamObject.InitSpawn called a yielding proc!"));
```